modules/gdscript/gdscript_parser.h

```diff
@@ -1283,6 +1283,8 @@ class GDScriptParser {
 	bool is_ignoring_warnings = false;
 	List<GDScriptWarning> warnings;
 	HashSet<GDScriptWarning::Code> ignored_warnings;
+	HashSet<int> script_ignored_warning_lines[GDScriptWarning::WARNING_MAX];
+	HashMap<GDScriptWarning::Code, int> script_ignored_warning_start_lines;
 	HashSet<int> unsafe_lines;
```

Comment:
I'm not sure if I used the correct structures.

Reply:
https://github.com/godotengine/godot/blob/c80a2b4fe99dcd0bba6fc24ed2748b1474b24448/core/templates/hash_map.h#L41-L45

Reply:
From what I read, Robin Hood swaps the entries, so if you do a `for each` on the hashmap, it's not guaranteed that you'll get each entry. If you're only getting the value that you want, it seems ok.